Documentation/templates/strings/objc.md

```diff
@@ -0,0 +1,53 @@
+## Template Information
+
+| Name      | Description       |
+| --------- | ----------------- |
+| File names | strings/objc-string-h.stencil & strings/objc-string-m.stencil |
+| Configuration example | <pre>strings:<br />  inputs: path/to/Localizable.strings<br />  outputs:<br />    - templateName: objc-string-h<br />      output: Localizable.h<br />    - templateName: objc-string-m<br />      output: Localizable.m</pre> |
```

Comment:
The config example seems wrong (wrong template names). Should be something like:
```yaml
strings:
  inputs: path/to/Localizable.strings
  outputs:
    - templateName: objc-h
      output: Localizable.h
    - templateName: objc-m
      output: Localizable.m
```

Reply:
(see above)